src/librepgp/stream-sig.h

```diff
@@ -292,10 +292,25 @@ typedef struct pgp_signature_t {
 
     /**
      * @brief Set the signer's uid, responcible for the signature creation. See the RFC
-     * 4880bis, 5.2.3.23 for details.
+     *        4880bis, 5.2.3.23 for details.
```

Comment:
Are these indentations above (6 changes) added intentionally or by mistake?

Reply:
@antonsviridenko yeah, those were added intentionally - as for me, description looks more clear in code when indented this way (and was wrapped by clang-format at some step before). However, if there is another opinion on readability we can discuss it further and update development documentation to have all documentation looks consistent.